jar.go

```diff
@@ -396,6 +396,34 @@ func (j *Jar) deleteExpired(now time.Time) {
 	}
 }
 
+// RemoveAll removes any cookies from the jar that were set for the given URL.
```

Comment:
This does not seem to take the URL path into account when deciding what to delete. That isn't unreasonable behavior, but let's document it. Perhaps change the parameter to host to make it clear what will be deleted.